rakelib/artifacts.rake

```diff
@@ -45,8 +48,10 @@ namespace "artifact" do
     end.flatten.uniq
   end
 
+  task "build-image" => ["bootstrap", "plugin:install-default"]
```

Comment:
yes, please help me choose the right name. I actually sat there for 15 minutes, brain fried and this is what I came up with :P